### DISCUSSION TITLE
Ansible error

### DISCUSSION THREAD

**Discussion by ArpanaM** [learndevopsonline/learndevopsonline#1915] [Q&A]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/68733492/45e500d6-5787-46fd-b807-db8079750ede)

When I am trying to install nginx am getting the following error. Please help!

**Comment by chaithanya1812** [comment]
Hi @ArpanaM, Execute this commands in ansible nodes(i.e, inv) servers.

sed -i -e "s|mirrorlist=|#mirrorlist=|g" /etc/yum.repos.d/CentOS-*
sed -i -e "s|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g" /etc/yum.repos.d/CentOS-*

**Reply by ArpanaM** [reply]
Thank you so much!

It got fixed
